easy_rec/python/main.py

```diff
@@ -341,11 +340,18 @@ def _train_and_evaluate_impl(pipeline_config,
   # Currently only a single Eval Spec is allowed.
   train_spec = tf.estimator.TrainSpec(
       input_fn=train_input_fn, max_steps=train_steps)
-  # create eval spec
-  eval_spec = _create_eval_export_spec(
-      pipeline_config, eval_data, check_mode=check_mode)
-  from easy_rec.python.compat import estimator_train
-  estimator_train.train_and_evaluate(estimator, train_spec, eval_spec)
+  if estimator_utils.has_sok():
+    estimator.train(
```

Comment:
why only call train when has_sok?

Reply:
evaluation under multi-worker mode is not good now, currently only works under single worker mode.